.github/workflows/validate_generated_files.yml

```diff
@@ -31,11 +29,11 @@ jobs:
       - name: Install web dependencies
         if: steps.cache.outputs.cache-hit != 'true'
         run: pnpm install --frozen-lockfile
-      - name: Generate zone config
-        if: steps.cache.outputs.cache-hit == 'true'
-        run: pnpm generate-zones-config
-      - name: Verify geometries
-        run: |
-          pnpm run update-world
+      - name: zones.json
+        run: pnpm run generate-zones-config
+        env:
+          VERIFY_NO_UPDATES: 1
```

Comment:
I think this generates the exchange config as well so we might want to rename it or split them up. Splitting them up would increase the accuracy of the tests.

Reply:
Oh yeah, good point! I think splitting them up would be good, but then we'd need to call two different scripts on "predevelop", "prebuild", etc., so for now I think it's easier to just rename :)

Reply:
We might not need two different files, we could do something like this in package.json and just split it into individual functions.
```json
"scripts": {
   "myScript": "node -e 'require(\"./someFile\").someFunction()'"
}
```
I have never used this myself though so not sure if/how it works.

Reply:
Sounds doable, but I'm also a bit afraid of what consequences this could have 😅 Let's add it to the backlog (which we're currently exploring where is)

Reply:
Sounds doable, but I'm also a bit afraid of what consequences this could have 😅 Let's add it to the backlog (which we're currently exploring where is)